src/eddymotion/utils.py

```diff
@@ -22,23 +26,20 @@
 #
 """Iterators to traverse the volumes in a 4D image."""
 
-import random
-from itertools import chain, zip_longest
-
```

Comment:
Revert the removal of imports and add the Generation import here:

```suggestion
import random
from itertools import chain, zip_longest
from typing import Generator
```